handlers/misttriggers/user_new.go

```diff
@@ -47,10 +46,17 @@ func ParseUserNewPayload(payload MistTriggerBody) (UserNewPayload, error) {
 func (d *MistCallbackHandlersCollection) TriggerUserNew(ctx context.Context, w http.ResponseWriter, req *http.Request, body MistTriggerBody) {
 	payload, err := ParseUserNewPayload(body)
 	cookies := req.Cookies()
-	accessKey := req.Header.Get("X-Livepeer-Access-Key")
-	jwt := req.Header.Get("X-Livepeer-JWT")
 
-	payload.Cookies = cookies // would remove probably when everything's working
+	var accessKey, jwt string
+	for _, cookie := range cookies {
```

Comment:
So is the new method using cookies or headers on the player side? From the PR description it looks like Mist grabs the header and puts it on the cookie for the `USER_NEW` request? Any reason to translate the header to a cookie? Or is this actually making access key/jwt work as a cookie on the player side? If the latter, please update the PR title/description to reflect that.

Reply:
On the player side, it's a header - mist then transform that header as a cookie and pass it in the USER_NEW trigger!

Related thread: https://discord.com/channels/423160867534929930/1171019521151225856/1176876688991330405